.github/workflows/stackhpc-ci-cleanup.yml

```
@@ -76,6 +76,26 @@ jobs:
           OS_APPLICATION_CREDENTIAL_ID: ${{ secrets.OS_APPLICATION_CREDENTIAL_ID }}
           OS_APPLICATION_CREDENTIAL_SECRET: ${{ secrets.OS_APPLICATION_CREDENTIAL_SECRET }}
 
+      - name: Clean up periodic multinode instances over 12 hours old
```

nit: not just periodic instances
```suggestion
      - name: Clean up GH actions multinode instances over 12 hours old
```